spec/rspec/core/resources/blocking_pipe_bisect_spec.rb_

```diff
@@ -0,0 +1,8 @@
+# Deliberately named *.rb_ to avoid being loaded except when specified
+
+RSpec.describe "1000 tests" do
+  puts "Try to saturate the pipe in Bisect command"
+  (0..1000).each do |t|
```

Comment:
Shouldn't it be 1548+?

Reply:
We run spec, that run a bisect command. When I did my test, I hit the cap  
with this amount of test. It pass with 922 occurrences, but not with 923.

With:
```ruby
  (0..923).each do |t|
    it { expect(t).to eq t }
  end
```
and `$ rspec --format doc` it get stuck here.

![Capture d’écran 2019-12-06 à 16 58 00](https://user-images.githubusercontent.com/8417720/70336618-bfeed100-1849-11ea-8299-5b03261ed446.png)

Reply:
I wonder if this is platform dependant and what we can do about that...

Reply:
Does it fail CI with 923?

Reply:
I tried on a VM with Linux ElementaryOS. I have the same result. With 923 we get stuck.

Reply:
I think we are good with the number 1000. Feel free to ask for more changes.

Reply:
I would really feel safer if it failed on CI without the fix. Otherwise looks good.

Reply:
Good idea. So we have the issue on most ruby with 923 tests https://travis-ci.org/rspec/rspec-core/builds/623575109
![Capture d’écran 2019-12-11 à 10 13 43](https://user-images.githubusercontent.com/8417720/70607629-1455d000-1bff-11ea-95b4-68405ce6b64b.png)

With 922 tests no stalled build https://travis-ci.org/rspec/rspec-core/builds/623575109

Reply:
Should we limit the test to OSX and Linux platform?

Reply:
Awesome!
You've linked the same (stalling) build, but I trust you on that one :D

I'd keep the test for all platforms, let's see if someone complains.

Reply:
Oups sorry:

923 tests: https://travis-ci.org/rspec/rspec-core/builds/623575109
922 tests: https://travis-ci.org/rspec/rspec-core/builds/623579071